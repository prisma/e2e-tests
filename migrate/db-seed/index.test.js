const { PrismaClient } = require('@prisma/client')

const client = new PrismaClient()

describe('prisma db seed', () => {
  it('successfully seeds the database', async () => {
    const users = await client.user.findMany()
    expect(users).toMatchSnapshot()
  })
})
