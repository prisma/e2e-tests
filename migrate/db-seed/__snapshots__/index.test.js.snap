// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prisma db seed successfully seeds the database 1`] = `
Array [
  Object {
    "email": "alice@prisma.io",
    "id": 1,
    "name": "Alice",
  },
  Object {
    "email": "nilu@prisma.io",
    "id": 2,
    "name": "Nilu",
  },
  Object {
    "email": "mahmoud@prisma.io",
    "id": 3,
    "name": "Mahmoud",
  },
]
`;
